### ISSUE TITLE
feeds are hard to find / access

### ISSUE THREAD

**Issue by vujevits** [agazso/postmodern#63]
### Ideas 
- Have a sidebar
- Search bar

**Comment by david-gauquelin** [comment]
I think this is a very high priority issue, because it prevents people from using the app. If I have nothing to follow the whole app is meaningless and I won't engage with it. I see multiple opportunities at the moment.

**Easier to find/add people I know**
- ability to search by name, email or username
- invite people to my feed by email or message (from my dashboard I can type an email or enter a phone number and we send an invite email with info to add that person (ideally with link to the app but not necessarily)

**Easier to access feeds**
At the moment the first 3 tabs in the app are confusing for me. I understand the concept of separating different types of content and I'm not necessarily saying that it's not a good idea but I think there multiple opportunities to make it better:
- ability to mark feeds as favorites directly from Settings/Feeds
- ability to view feed from Settings/Feeds (not only the QR code)
- ability to view/manage a list of corresponding feeds directly from the News or Favorite page (view, remove, add, jump to...)

**Questions**
I'm not sure if the bottom tabs are the right place to separate feed, as an early idea I'm thinking that it might be a better idea to have a single feed tab and have a simple sub-menu/filter to isolate those feeds. Maybe we can even imagine combining more than just 1 category at a time. I would explore this idea

I'm not sure if it's a good idea to keep my posts in the first tab. I would rather expect to access this from my profile or something like this. It doesn't seem to be the most important thing and it's given the default, most prominent place atm.

**Comment by agazso** [comment]
I agree this is very important. We have an onboarding label that we use for the issues that affects the users' first experience with the app. I added it to this issue.

I also created a new issue regarding making finding and adding people you know. Let's separate that discussion and in this issue focus on the feeds only.
https://github.com/agazso/postmodern/issues/139

We also already worked on the issues you mention (ability to mark feeds as favorites, ability to view feed from Settings/Feeds) and they are already merged and will be released in the next version (probably early this week).
https://github.com/agazso/postmodern/pull/127

So the current situtation with the tabs are definitely a result of an organic evolution and is not meant to be a final solution, just an experiment to try out the separation of different feeds until we build a filter component. So that seems like the logical next step to work on, because you independently reached that conclusion too.

The my posts are the first tab is also a result of how I started using the app when there weren't even sharing: basically I started collecting my favorite photos and recording important moments from my life. Then the idea of different feeds came from that I don't want to share all of this with everyone, so how is it be possible to share it selectively. Some of it I don't even want to share with anyone (or not now).

To me this is one of the nicest thing in the app, that it's the place where I store important events from my life and can look back. All the while I can be sure that my privacy is respected. So in my use-case putting it under the profile would mean that it's a bit hidden and hard to access, because normally you only go in your profile when you want to do some changes in settings.
